src/types.rs

```diff
@@ -78,18 +98,11 @@ impl fmt::Display for Task {
             write!(f, " {}", tag)?;
         }
         if self.priority != 0 {
-            let mut prio: String = String::new();
-            for i in 0..self.priority {
-                prio.push(',');
-            }
-            write!(f, " [{}]", prio)?;
+            write!(f, " .{}", self.priority)?;
+        }
+        if self.finished {
+            write!(f, " FINISHED")?;
         }
         write!(f, "")
```

Comment:
nit: this call seems doesn't really do anything. I think this can be safely removed.

Reply:
It cant, its purely there to indicate if a task is marked as finished, the call in line 106 is there to have the correct return type

Reply:
Ah, yes, you're right. My bad.